### PR TITLE
RN Jan 27

### PR DESCRIPTION
**Pull request by jshackell-sfdc**

### CODE REVIEW

File: releasenotes/sfdx/README.md [hunk 1]
```diff
@@ -14,19 +14,35 @@ If you use [autocomplete](https://developer.salesforce.com/docs/atlas.en-us.sfdx
...
+## 7.136.0 (Feb 3, 2022) [stable-rc]
 
 These changes are in the Salesforce CLI release candidate. We plan to include these changes in next week's official release. This list isn't final and is subject to change.
 
+* NEW: Get shorter JSON output from running the `force:mdapi:beta:deploy` or `force:mdapi:beta:deploy:report` commands by using the new `--concise` parameter with the `--json` parameter. The new parameter omits success messages from the JSON output. When the two beta commands become generally available, the GA commands, such as `force:mdapi:deploy`, will also have the new parameter. 
+
+* CHANGE: The [Salesforce CLI open-sourcing train](https://developer.salesforce.com/blogs/2021/02/open-sourcing-salesforce-cli-update-feb-2021) is speeding along and this week we've created the `force:org:beta:create` beta command in the [plugin-org](https://github.com/salesforcecli/plugin-org) plug-in. The command works the same as its equivalent existing one. Try out this beta command before we make the open-source version of `force:org:retrieve` generally available.
```
Comment (mshanemc) [inline releasenotes/sfdx/README.md]: there's no `org:retrieve`

**Review by mshanemc — CHANGES_REQUESTED** [review]
there's also several closed WI I see with `53.16.0` on them that didn't appear (emailTemplateFolder fix, more gitignore stuff, metadata rest auth bug)

File: releasenotes/sfdx/README.md (hunk 1 above)
Comment (jshackell-sfdc) [reply releasenotes/sfdx/README.md]: yeah, i know, i thought I'd fixed that...

Comment (jshackell-sfdc) [reply releasenotes/sfdx/README.md]: @mshanemc , those bugs don't have Doc Status set to New :)   should all of them be doc'd?

**Pull request merged**